projects/plugins/jetpack/_inc/client/sharing/publicize.jsx

```diff
@@ -71,12 +88,56 @@ export const Publicize = withModuleSettingsFormHelpers(
 								link: getRedirectUrl( 'jetpack-support-publicize' ),
 							} }
 						>
-							<p>
-								{ __(
-									'Connect your website to the social media networks you use and share your content across all your social accounts with a single click. When you publish a post, it will appear on all connected accounts.',
-									'jetpack'
-								) }
-							</p>
+							<p>{ jetpackSocialText }</p>
+							{ showUpgradeLink && (
+								<React.Fragment>
+									<p>
+										{ ! hasSocialBasicFeatures
+											? createInterpolateElement(
+													__(
+														'<moreInfo>Upgrade to a Jetpack Social plan</moreInfo> to get unlimited shares and advanced media sharing options.',
+														'jetpack'
+													),
+													{
+														moreInfo: (
+															<a
+																href={ getRedirectUrl(
+																	'jetpack-plugin-admin-page-sharings-screen',
+																	{
+																		site: siteRawUrl,
+																		query:
+																			'redirect_to=' +
+																			encodeURIComponent( window.location.href.split( '#' )[ 0 ] ),
+																	}
+																) }
+															/>
+														),
+													}
+											  )
+											: createInterpolateElement(
+													__(
+														'<moreInfo>Upgrade to the Jetpack Social Advanced plan</moreInfo> to get advanced media sharing options.',
+														'jetpack'
+													),
+													{
+														moreInfo: (
+															<a
+																href={ getRedirectUrl(
+																	'jetpack-plugin-admin-page-sharings-screen',
+																	{
+																		site: siteRawUrl,
+																		query:
+																			'redirect_to=' +
+																			encodeURIComponent( window.location.href.split( '#' )[ 0 ] ),
```

Comment:
Could we use `siteAdminUrl` for those links?

Reply:
I have done that, but I needed to strip off the last slash in it. This is because, on the pricing page, the final URL looks like this `https://cloud.jetpack.com/pricing/?site=sidtest.jurassic.tube&view=products&redirect_to=<redirect-url>#jetpack_social_advanced_yearly` We need the `#jetpack_social_advanced_yearly` bit in the end to open the pricing modal. 

If I did not strip off the trailing slash from siteAdminUrl, `jetpack_social_advanced_yearly` is seen as a part of the redirect URL and the pricing modal doesn't open up.